LiveSplit.AutoSplitters.xml

```diff
@@ -21913,13 +21913,13 @@
     </AutoSplitter>
     <AutoSplitter>
         <Games>
-            <Game>Awesome Tanks</Game>
+            <Game>Marble Mania</Game>
         </Games>
         <URLs>   
-            <URL>https://raw.githubusercontent.com/Biksell/asl/main/Awesome%20Tanks/AwesomeTanks.asl</URL>
+            <URL>https://github.com/cyanbeast01/Marble-Mania/blob/main/Marble-Mania-livesplliter(buggy).asl</URL>
```

Comment:
This needs to be a raw URL. Also, if it's buggy, then don't add it here.

Reply:
i guess